### PR TITLE
Removal of mocha-eslint and eslint-config-node-services deps.

### PR DESCRIPTION
**Pull request by hknustwmf**
Minor formatting updates. Added .eslintrc for style exceptions

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -51,18 +52,15 @@
...
     "js-yaml": "^3.11.0",
     "nock": "^9.2.5",
     "preq": "^0.5.6",
     "ajv": "^5.5.2",
-    "mocha-eslint": "^4.1.0",
-    "eslint": "^4.19.1",
-    "eslint-config-node-services": "^2.2.5",
-    "eslint-config-wikimedia": "^0.5.0",
-    "eslint-plugin-jsdoc": "^3.6.3",
+    "eslint": "^5.12.0",
```
Comment (Pchelolo) [inline package.json]: I believe the eslint-config-wikimedia already depends on eslint, no need to explicitly add this dep here.

File: package.json [hunk 2]
```diff
@@ -51,18 +52,15 @@
...
+    "eslint-config-wikimedia": "^0.10.0",
+    "eslint-plugin-jsdoc": "^4.0.0",
```
Comment (Pchelolo) [inline package.json]: I think we might want to upstream this dep to the config.

Comment (hknustwmf) [reply package.json]: plugin-jsdoc? Is that something I need to do? Or hold off on that for now?

Comment (Pchelolo) [reply package.json]: Nothing to do now, just a mental note for when we will be upstreaming some of the overrides

**Pull request merged**